### PR TITLE
Blacklist invalid raw data

### PR DESCRIPTION
**Pull request by lvonasek**
improvement

Blacklisting filtered scans with an error higher than 10cm but also some valid scans with low error (that's the reason the numbers changed in both directions). However overall average error decreased significantely.

Note that height predictions parameters were adjusted.

There were blacklisted 66 of total 1306 standing children scans. None of lying children scans was blacklisted

### CODE REVIEW

**Review by prajurock — APPROVED** [review]

**Comment by prajurock** [comment]
LGTM

File: cgmml/common/evaluation/CV/rgbd_match.py [hunk 1]
```diff
@@ -1,11 +1,64 @@
 import sys
```
[the remark this answers is not in the document]
Comment (lvonasek) [reply cgmml/common/evaluation/CV/rgbd_match.py:1]: How should it be sorted? Now it is common imports first and afterwards cgm-specific imports.

Comment (prajurock) [inline cgmml/common/evaluation/CV/rgbd_match.py:1]:
```suggestion
import sys

```

**Review by prajurock — APPROVED** [review]

File: cgmml/common/evaluation/CV/rgbd_match.py (hunk 1 above)
Comment (prajurock) [reply cgmml/common/evaluation/CV/rgbd_match.py:1]: @lvonasek

**Pull request merged**